lib/emoji.js

```diff
@@ -63,14 +81,25 @@ Emoji.prototype.which = function which(emoji_code) {
 /**
  * emojify a string (replace :emoji: with an emoji)
  * @param  {string} str
+ * @param  {object} options
  * @return {string}
  */
-Emoji.prototype.emojify = function emojify(str) {
+Emoji.prototype.emojify = function emojify(str, options) {
+  options = options ? options : {};
   var emoji = this;
-  return str.split(Emoji.parser) // parse emoji via regex
+  var result = str.split(Emoji.parser) // parse emoji via regex
             .map(function parseEmoji(s, i) {
               return (i % 2 === 0) ? s : emoji._get(s); // every second element is an emoji, e.g. "test :fast_forward:" -> [ "test ", "fast_forward" ]
             })
             .join('') // convert back to string
+
```

Comment:
Yes that was my first shot ;) But I've changed due to the incomplete `stringEmojiParser`. I'll work on that